tools/README.md

```diff
@@ -36,6 +36,34 @@ Build openpilot with this command:
 scons -u -j$(nproc)
 ```
 
+### Dev Container
+
+openpilot supports [Dev Containers](https://containers.dev/). Dev containers provide customizable and consistent development environment wrapped inside a container. This means you can develop in a designated environment matching our primary development target, regardless of you local setup.
+
+Dev containers are supported in [multiple editors and IDEs](https://containers.dev/supporting), including Visual Studio Code. To start using it in VS Code, make sure you have `docker` installed. Then install `Dev Containers` extension. More detailed installation instructions can be found [here](https://code.visualstudio.com/docs/devcontainers/containers#_installation).
```

Comment:
this new section is quite long. we should just link to the real instructions for each thing, i.e. https://code.visualstudio.com/docs/devcontainers/create-dev-container for this one